scripts/safe_template_linter.py

```diff
@@ -1883,6 +1887,12 @@ def _get_contexts(self, mako_template):
                 contexts.append({'index': context.start(), 'type': 'html'})
             elif match_string.startswith("<%static:require_module"):
                 contexts.append({'index': context.end(), 'type': 'javascript'})
+            elif match_string.startswith("</%static:require_module>"):
+                contexts.append({'index': context.start(), 'type': 'html'})
+            elif match_string.startswith('<%block name="requirejs"'):
```

Comment:
The regex allowed single-quotes around the name also.  You can move all this into the regex by using group names, and then asking which group name matched, so that you don't have to duplicate the text from the regex in the tests.  Not sure if it would be nicer in the end, but it could be worth a look.

match.lastgroup is the name of the last group matched.